### PR TITLE
LVM-463 add attributes (City, State, Zipcode) to Affiliate 

### PR DESCRIPTION
**Pull request by jlally21**
https://cs500-jira.ccs.neu.edu/browse/LVM-463

requires Updates to all Tests, as well as MVC for Affiliate

affiliate edit view

### CODE REVIEW

File: spec/models/affiliate_spec.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@
           .is_at_most(255)
       end
 
-      it 'email formate' do
+      it 'email format' do
```
Comment (zack) [inline spec/models/affiliate_spec.rb:33]: 👌

File: spec/views/affiliates/index_spec.rb [hunk 2]
```diff
@@ -2,29 +2,14 @@
...
     render
-    assert_select 'tr>td', text: 'Name'.to_s, count: 2
-    assert_select 'tr>td', text: 'Address'.to_s, count: 2
-    assert_select 'tr>td', text: 'PhoneNumber'.to_s, count: 2
-    assert_select 'tr>td', text: 'Email@1.com'.to_s, count: 1
-    assert_select 'tr>td', text: 'Email@2.com'.to_s, count: 1
-    assert_select 'tr>td', text: 'Website'.to_s, count: 2
-    assert_select 'tr>td', text: 'Twitter'.to_s, count: 2
+    expect(rendered).to match(@affiliate.name)
+    expect(rendered).to match(@affiliate2.name)
```
Comment (zack) [inline spec/views/affiliates/index_spec.rb:13]: nice file updates

**Review by zack — APPROVED** [review]
just do a `rake db:reseed` and force push back up again to resolve that merge conflict and i think you'll be good to go

**Comment by zack** [comment]
also please remember to squash + merge!

**Review by nabeel- — APPROVED** [review]

**Pull request merged**